api/spec/spec_handler.go

```diff
@@ -894,6 +907,9 @@ func (d *specHandler) SpecOptsFromString(
 	if ok, fsFormatOptions := d.getVal(SpecFsFormatOptionsRegex, str); ok {
 		opts[api.SpecFsFormatOptions] = fsFormatOptions
 	}
+	if ok, shared := d.getVal(sharedBlockRegex, str); ok {
+		opts[api.SpecSharedBlock] = shared
+	}
```

Comment:
There is spec_handler_test.go, can we add unit tests there as well?

Reply:
added UT